LandSandBoat/server
# [dbtool] Refactor to allow running from other directories like repo root

**_I affirm:_**

## What does this pull request do?

As part of https://github.com/LandSandBoat/server/issues/2240, one of the issues is to remove the requirement for dbtool to always be run from <root>/tools. This does that.

## Steps to test these changes

Use dbtool as normal, but launch it from `<root>/tools`, or `<root>`, or `<root>/scripts/globals` and make sure everything works as expected.
